Cannot create vsix file due to src/test/runTest.ts

**Describe the bug**
when running `vsce package` you get the following error:
```
ERROR in VSCode-Couchbase/src/test/runTest.ts(19,26)
      TS2307: Cannot find module 'vscode-test' or its corresponding type declarations.
```
This prevents a vsix file from being generated

**Steps to reproduce**
Navigate to the main branch and run `vsce package`

**What did you expect to see?**
A vsix file should be produced

**What did you see instead?**
no vsix file

**What version did you use?**
VSCode Version: 1.70.1 (Universal)
Extension Version: v0.4.0 - note I commented out the problematic code to make the 0.4.0 release but it should still be in the github repo

--- a/src/test/runTest.ts
+++ b/src/test/runTest.ts
@@ -16,7 +16,7 @@
 
 import * as path from "path";
 
-import { runTests } from "vscode-test";
+import { runTests } from "@vscode/test-electron";
 
 async function main() {
   try {

